kuka_kr5_support/urdf/kr5_arc.xacro

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<!--Generates a urdf from the macro in kr16_2_macro.xacro -->
```

Comment:
This can probably be removed: references the KR 16 and this xacro is so simple it's quite clear what it does.